app/views/search/_filter.html.erb

```diff
@@ -15,9 +15,11 @@
           <a href="<%= results_path(add_filter(@enhanced_query, category, term['key'])) %>">
           <span class="sr">Apply filter:</span>
         <% end %>
-          <% if Flipflop.enabled?(:gdt) %>
-            <span class="name"><%= gdt_sources(term['key'], category) %></span>
-          <% end %>
+        <% if Flipflop.enabled?(:gdt) %>
+          <span class="name"><%= gdt_sources(term['key'], category) %></span>
+        <% else %>
```

Comment:
🤣 🙈 

Yeah, that indeed is what we are seeing :) 

I'm now sort of wondering if `gdt_sources` is a misleading method name as I sort of expect that to only affect the `source` filter and not all filters. I feel that not strongly enough to suggest changing it as part of this fixup.

Reply:
I struggled with that, too. We decided to refactor `gdt_sources` to return the value as-is if it's not a source as a safety measure, but it may be more readable to remove that from the helper method and go back to something like `gdt_sources(term['key'], category) || term['key']`? 🤷